indico/modules/events/abstracts/templates/display/abstracts.html

```diff
@@ -0,0 +1,25 @@
+{% extends 'layout/conference_page_base.html' %}
+
+{% block title %}
+    {% trans %} Abstracts from {% endtrans %} <span class="track-name">{{ track_title }}</span>
```

Comment:
The last part should be inside the trans - otherwise it'd look awful in any language where the title isn't at the very end